### PR TITLE
Add sample 5 - running docker container based batch task.

### PR DESCRIPTION
**Pull request by YidingZhou**

### CODE REVIEW

**Comment by matthchr** [comment]
@YidingZhou Waiting for an update on codeflow CR comment, then will merge.

**Comment by YidingZhou** [comment]
@matthchr updated.

File: Python/Batch/sample5_docker_batch_task.py [hunk 1]
```diff
@@ -139,7 +139,7 @@ def add_docker_batch_task(batch_client, block_blob_client, job_id, pool_id):
     :param str job_id: The id of the job to use.
     :param str pool_id: The id of the pool to use.
     :rtype: str
```
Comment (matthchr) [inline Python/Batch/sample5_docker_batch_task.py]: This should be `list` I think

**Comment by YidingZhou** [comment]
@matthchr updated. btw, we should update all python samples - replace CloudPool w/ PoolAddParameter for pool creations, plus job/task.

File: Python/Batch/README.md [hunk 2]
```diff
@@ -64,6 +64,16 @@ tunnel script to interact with the batch pool after the sample is run. When
 using this option, you will also need to disable the delete pool option as
 well.
 
+####[sample5\_docker\_batch\_task.py](./sample4\_docker\_batch\_task.py)
+This sample shows how to schedule tasks that run a docker container based
+task. Specifically, this samples use pool start up task to install docker
```
Comment (matthchr) [inline Python/Batch/README.md]: "this sample uses a pool start up task to install docker on the Batch VMs." ?

File: Python/Batch/README.md [hunk 3]
```diff
@@ -64,6 +64,16 @@ tunnel script to interact with the batch pool after the sample is run. When
...
+on Batch VMs which is Ubuntu based. It then uses job preparation tasks to
+pull the application image from docker hub and finally submit a set of simple
+tasks. Each task will launch a docker container to perform ffmpeg transcoding
+on a downloaded mp4 file and upload the result to blob storage. The docker
+image is CentOS based with ffmpeg and blobxfer preinstalled. The docker image
+(yidingz/ffmpeg:v3) is available on docker hub and downloadable to everyone.
```
Comment (matthchr) [inline Python/Batch/README.md]: Just remove the "and downloadable to everyone" bit

File: Python/Batch/common/helpers.py [hunk 4]
```diff
@@ -265,6 +265,29 @@ def wait_for_all_nodes_state(batch_client, pool, node_state):
         time.sleep(10)
 
 
+def create_container_and_create_sas(
```
Comment (matthchr) [inline Python/Batch/common/helpers.py]: Why do you need this helper method?  It seems to duplicate the existing `create_sas_token` at the very least it should use that method instead -- but I'd prefer to see it just removed and in your code you can do:

```
block_blob_client.create_container(
    container_name,
    fail_on_exist=False)
helpers.create_sas_token(block_blob_client, container_name, blob_name, permission, expiry)
```

File: Python/Batch/configuration.cfg [hunk 5]
```diff
@@ -1,9 +1,9 @@
...
-storageaccountname=
-storageaccountkey=
+[Batch]
+batchaccountname=
+batchaccountkey=
+batchserviceurl=
+
+[Storage]
+storageaccountname=
+storageaccountkey=
```
Comment (matthchr) [inline Python/Batch/configuration.cfg]: Why was this file changed at all?

**Comment by matthchr** [comment]
@YidingZhou I will fix the other Python samples here shortly.
Also, added a few remaining comments -- sorry for giving them so peacemeal.

**Comment by YidingZhou** [comment]
Updated.
